JobConfig/reco/IPA-Michel.fcl

```diff
@@ -16,25 +16,25 @@ physics.producers.KSFDeM.ParameterErrors : [20.0, 0.1, 0.01, 20.0, 0.1]
 # find low-momentum helices 
 physics.producers.HelixFinderDe.targetconsistent_init : false
 physics.producers.HelixFinderDe.targetconsistent : false
-physics.producers.HelixFinderDe.RobustHelixFit.minCenterR : 300
-physics.producers.HelixFinderDe.RobustHelixFit.maxCenterR : 500
-physics.producers.HelixFinderDe.RobustHelixFit.minR : 100
-physics.producers.HelixFinderDe.RobustHelixFit.maxR : 200
-physics.producers.HelixFinderDe.RobustHelixFit.minAbsLambda : 50
-physics.producers.HelixFinderDe.RobustHelixFit.maxAbsLambda : 150
-physics.producers.HelixFinderDe.RobustHelixFit.minzsep : 40
-physics.producers.HelixFinderDe.RobustHelixFit.maxzsep : 200
-physics.producers.HelixFinderDe.RobustHelixFit.mindist : 50
-physics.producers.HelixFinderDe.RobustHelixFit.maxdist : 250
-physics.producers.HelixFinderDe.RobustHelixFit.minArea : 800
-##physics.producers.HelixFinderDe.minNHit : 40
+physics.producers.HelixFinderDe.HelixFitter.minCenterR : 300
+physics.producers.HelixFinderDe.HelixFitter.maxCenterR : 500
+physics.producers.HelixFinderDe.HelixFitter.minR : 100
+physics.producers.HelixFinderDe.HelixFitter.maxR : 200
+physics.producers.HelixFinderDe.HelixFitter.minAbsLambda : 50
+physics.producers.HelixFinderDe.HelixFitter.maxAbsLambda : 150
+physics.producers.HelixFinderDe.HelixFitter.minzsep : 40
+physics.producers.HelixFinderDe.HelixFitter.maxzsep : 200
+physics.producers.HelixFinderDe.HelixFitter.mindist : 50
+physics.producers.HelixFinderDe.HelixFitter.maxdist : 250
+physics.producers.HelixFinderDe.HelixFitter.minArea : 800
+physics.producers.HelixFinderDe.HelixFitter.minNHit : 1
```

Comment:
We should consider if CalPatRec needs equivalent changes for reconstructing IPA electrons.  I don't know if anyone has studied that.  It doesn't need to be part of this PR

Reply:
Yes, good point, we are beginning to look at this in more detail since as we spoke about before these parameters dont help as much as they used to in terms of efficiency increase